portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,22 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+//Causes the bubble to slide down the page.
+function bubbleFall() {
+    //Find the bubble.
+    var elem = document.getElementById('bubble');
+    var pos = 0;
+    //Make the mobement visible and fluid by moving every 10 milliseconds;
+    var id = setInterval(frame, 10)
+    //For each movement, move the element one pixel down. At the bottom, stop moving.
+    function frame(){
```

Comment:
Please add a space after the () before the {, here and below.

Reference: https://google.github.io/styleguide/jsguide.html#formatting-horizontal-whitespace